ratis-common/src/main/java/org/apache/ratis/protocol/DataStreamRequest.java

```diff
@@ -19,7 +19,9 @@
 package org.apache.ratis.protocol;
 
 import org.apache.ratis.io.WriteOption;
+import java.util.List;
 
 public interface DataStreamRequest extends DataStreamPacket {
-  WriteOption[] getWriteOptions();
```

Comment:
This is a public interface.  We must first deprecate it before removing it later.